bundler/lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -41,7 +37,7 @@ Gem::Specification.new do |spec|
   spec.extensions = ["ext/<%= config[:underscored_name] %>/extconf.rb"]
 <%- end -%>
 <%- if config[:ext] == 'rust' -%>
-  spec.extensions = ["ext/<%= config[:underscored_name] %>/Cargo.toml"]
+  spec.add_dependency "rb_sys", "~> 0.9"
```

Comment:
The `CargoBuilder` was a good first start, but it's much less flexible and not the best default IMO. rb-sys + create_rust_makefile is much more robust these days, so let's use it